quarto/table-templates/template-table_32.qmd

```diff
@@ -40,6 +40,64 @@ Required variables:
 -   **`advs`**: `USUBJID`, `AVISITN`, `PARAMCD`, `AVAL`, `AVALU`, and the variables specified by `arm_var` and `saffl_var`.
 -   **`alt_counts_df`** (if specified): `USUBJID` and the variables specified by `arm_var` and `saffl_var`.
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+| Argument         | Description                                                                                                                                                                                                                                            | Default      |
+|:--------------|:------------------------------------------|:--------------|
+| `advs`           | (`data.frame`) Dataset (typically ADVS) required to build table.                                                                                                                                                                                       | *No default* |
```

Comment:
The function requires the `adsl` dataset as well I believe. So this should be included.